pkg/proc/target_group.go

```diff
@@ -0,0 +1,125 @@
+package proc
+
+import (
+	"fmt"
+	"strings"
+)
+
+// TargetGroup reperesents a group of target processes being debugged that
+// will be resumed and stopped simultaneously.
+// New targets are automatically added to the group if exec catching is
+// enabled and the backend supports it, otherwise the group will always
+// contain a single target process.
+type TargetGroup struct {
+	targets []*Target
+	procs   []ProcessInternal
+	Sel     *Target
```

Comment:
nit: lets replace `Sel` with something more descriptive like `Main` or `Root` (as in root of the process tree) or just don't abbreviate and use `Selected`.

Reply:
Changed to Selected (it's meant to be the one on which debugger operates currently).